tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -539,7 +542,7 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
               visible: Boolean(
                 runSelectionMap &&
                   runSelectionMap.get(runId) &&
-                  renderableRuns.has(runId)
+                  (renderableRuns.has(runId) || !hparamsInTimeSeriesEnabled)
```

Comment:
Perhaps put the flag first?
`(!hparamsInTimeSeriesEnabled || renderableRuns.has(runId))`

Personally, I think this would be the most readable, but some might object to it as being unnecessarily verbose:
`(hparamsInTimeSeriesEnabled ? renderableRuns.has(runId) : true)`